tableauserverclient/models/target.py

```diff
@@ -0,0 +1,14 @@
+"""Target class meant to abstract mappings to other objects"""
+class Target():
```

Comment:
Two blank lines here (between docstring and the class def) should fix the style checker
`tableauserverclient/models/target.py:2:1: E302 expected 2 blank lines, found 0`